priv/template/mix.exs.eex

```diff
@@ -25,7 +25,9 @@ defmodule <%= @module %>.Mixfile do
   defp deps do
     [
       {:ace, "~> 0.18.0"},
-      {:raxx_static, "~> 0.7.0"},<%= if @exsync do %>
+      # temporary while working on middleware branches
+      {:raxx, github: "crowdhailer/raxx", branch: "release-1.0", override: true},
```

Comment:
This shouldn't be necessary I can make a version of raxx_static that uses the current version of Raxx